app/flatpak-builtins-build-bundle.c

```diff
@@ -177,6 +173,11 @@ iterate_bundle_icons (GFile                     *root,
   g_autoptr(GFile) icons_dir =
     g_file_resolve_relative_path (root,
                                   "files/share/app-info/icons/flatpak");
+  if (!g_file_test (flatpak_file_get_path_cached (icons_dir), G_FILE_TEST_IS_DIR)) {
```

Comment:
Don't mix code in the middle of the variable definitions like this.

Also, don't use flatpak_file_get_path_cached() here, because the files passed in are not always regular files, but rather OstreeRepoFile objects that implement the GFile interface. This is why the tests are breaking.

Something like g_file_query_file_type (icons_dir, 0, NULL) != G_FILE_TYPE_DIRECTORY) should work.

Reply:
Also, brackets on new line.

Reply:
This fixes the tests, but testlibrary timesout still